### DISCUSSION TITLE
Theme per filetype

### DISCUSSION THREAD

**Discussion by hfn92** [NvChad/NvChad#2612] [Q&A] [answered]
Is it possible to use a different theme depending on the filetype? I tried some plugings, but I guess they don't work with NvChads builtin themes.

**Comment by siduck** [comment]
not possible yet :/

**Reply by hfn92** [reply, marked as answer]
I see, thanks

I was able to achieve what I want using  ts highlights e.g `@type.qualifier.cpp`, Though the language specific ones didn't work for me with `polish_hl` and I had to use `nvim_set_hl`.

Thanks for your response :+1:

**Reply by siduck** [reply]
@hfn92  use hl_add for that , cuz its a new hlgroup
```lua
hl_add = {

["@comment.lua"] =  { fg = 'red' },
["@comment.python"] = { bg = 'pink' }

}
